src/main/kotlin/no/nav/familie/ef/sak/repository/domain/BehandlingJournalpost.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.Table
+
+@Table("behandling_journalpost")
+data class BehandlingJournalpost(@Column("journalpost_id") val journalpostId: String,
```

Comment:
Er det mulig å legge på et felt for om journalposten er inngående eller utgående? Kan være fint å skille på feks om vi vil vise alle inngående dokumenter knyttet til en behandling.

Reply:
Godt poeng. Bør vi bruke enumen "JournalpostType" fra familie.kontrakter, lage egen eller bare bruke "string" da?

Føles litt "uriktig" å bruke enum fra kontrakter i databasen, men det er kanskje bare vanesak?